content/quickstart.md

```diff
@@ -345,3 +345,97 @@ and sends invoices to clients on the same schedule.
 
 [harvest]: https://www.getharvest.com/
 [accounts]: mailto:accounts@oddbird.net
+
+## Success Criteria:
+
+This guide will help you understand the expectations for your role at OddBird.
+The criteria are divided into three groups: Work Quality, Communication, and
+Learning & Growth. Each group has three levels of performance: Underperforms,
+Meets Expectations, and Exceeds Expectations. You can use this guide to assess your own performance,
+identify your strengths and areas for improvement, and set your goals and action
+plans.
+
+### Work Quality
+
+#### Underperforms
+
+- Lacks initiative and waits for direction or assignments on high priority work
+- Produces low-quality or incomplete work that requires excessive rework or
+  refactoring from others
+- Uses outdated or inappropriate tools, technologies, or methodologies
+- Avoids or gives up on challenges
+- Does not plan or dissect big assignments into smaller tasks
+
+#### Meets Expectations
+
+- Takes initiative on high priority work by chipping away at it
+- Delivers complete and high-quality work, and acts on feedback from others
+- Approaches challenges by spending time planning and dissecting out assignments
+
+#### Exceeds Expectations
+
+- Takes ownership of features and infrastructure
+- Leads internal review processes and delivers high-quality, actionable feedback
+  to others
+- Makes requests for input early in process to tackle challenging work
+- Takes initiative and leads others in planning and breaking down upcoming work
+
+### Communication
+
+#### Underperforms
+
+- Rarely communicates or does so in an untimely manner
+- Misses large parts of or entire meetings without notice
+- Does not document the breakdown of upcoming work or the status of in-progress
+  work, and does not reflect on their contributions, accomplishments, or lessons
+  from challenges (self-retro)
+- Does not ask for help or guidance when blocked, and does not participate or
+  learn from review and discussions
+- Shows disinterest or indifference when planning new work
+
+#### Meets Expectations
+
+- Communicates frequently and in a timely manner
+- Attends most meetings and notifies others in advance when they will be late or cannot attend
+- Posts a breakdown of their daily assignments documenting completed work, the
+  status of in-progress work, and upcoming tasks
+- Documents their contributions, accomplishments, and lessons from challenges
+  (self-retro)
+- Reaches out with questions when blocked and engages with review and
+  planning by asking follow-up questions
+
+#### Exceeds Expectations
+
+- Communicates with trust in advance and requests engagement from others
+- Leads meetings and discussions about internal and external projects
+- Follows up during and after reviews and discussions with questions that help
+  confirm understanding
+- Generates internal documents, blog posts, and other valuable content based on
+  their contributions (especially accomplishments and challenges)
```

Comment:
Perhaps covered in other points throughout, but another area of exceeding expectations would be challenging and defining process, and not just following existing process. Perhaps something like-

Meets Expectations-
`Challenges the team processes and looks for ways to improve them`

Exceeds Expectations-
`Adjusts the team processes, listening to feedback and guiding the team through the changes`

These are mostly pulled from http://www.engineeringladders.com/

Reply:
Great link! I added both and another entry to "Underperform" in baec15edfa062d3ac7c7ea22c86a4f0cf645f8bf